Workaround for Perl regex deprecation

Perl 5.21.1 throws warnings for unescaped '{', and pari treats
warnings as errors by default. Patch taken from

https://projects.archlinux.org/svntogit/community.git/tree/trunk/no_warnings_fatal.patch?h=packages/pari-sage



--- src/src/desc/doc_make.orig	2015-07-04 11:03:38.247781017 +0200
+++ src/src/desc/doc_make	2015-07-04 11:03:53.224517403 +0200
@@ -1,5 +1,4 @@
 #!/usr/bin/perl
-use warnings FATAL => 'all';
 use strict;
 use PARI::822;
 
